sdk/src/wallet/account/builder.rs

```diff
@@ -128,7 +123,7 @@ impl AccountBuilder {
                 let bech32_hrp = {
                     match bech32_hrp {
                         Some(bech32_hrp) => bech32_hrp,
-                        None => client.get_bech32_hrp().await?,
+                        None => self.wallet.client.get_bech32_hrp().await?,
```

Comment:
and here

Reply:
and a few others where we call direcy, ill tag em if we want to change

Reply:
This is called on the AccountBuilder not Account, so that method do not exist.

Reply:
Nevermind, I'm wrong lol